desktop/src/main/java/bisq/desktop/main/MainView.java

```diff
@@ -809,6 +809,10 @@ private void setupBadge(JFXBadge buttonWithBadge, StringProperty badgeNumber, Bo
 
             this.setToggleGroup(navButtons);
             this.getStyleClass().add("nav-button");
+            // Japanese fonts are dense, increase top nav button text size
```

Comment:
Why is this change part of this PR? I think something got wrong during rebasing of your branch.